src/manifolds/Circle.jl

```diff
@@ -211,7 +228,7 @@ mean modulo 2π.
 mean(::Circle, ::Any)
 mean(::Circle,x::Array{<:Real},w::AbstractVector; kwargs...) = sym_rem(sum(w.*x))
 
-@inline norm(::Circle, x, v) = sum(abs.(v))
+@inline norm(::Circle, x, v) = sum(map(abs, v))
```

Comment:
Should be able to do this
```suggestion
@inline norm(::Circle, x, v) = sum(abs, v)
```